web-client/src/pages/signup.js

```diff
@@ -0,0 +1,65 @@
+import '../components/signup.css';
+import '../global.css';
+
+export default function Signup(){
+    return (
+        <>
+        <header className="site-header">
+      <h1><a href="index.html" className="site-header-heading">FSWD</a></h1>
+    </header>
+    <main>
+      <nav aria-label="breadcrumb" className="breadcrumb">
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><span aria-current="page">Sign-up</span></li>
+        </ul>
+      </nav>
+      <div id="main-container">
+        <form action="https://jsonplaceholder.typicode.com/users" method="POST" id="sign-up-form">
+          <div className="form-group">
+            <label for="sign-up-form-name">Name</label>
+            <input type="text" id="sign-up-form-name" name="name" required/>
+          </div>
+          <div className="form-group">
+            <label for="sign-up-form-email">Email</label>
+            <input type="email" name="email" required id="sign-up-form-email"/>
+          </div>
+          <div className="form-group">
+            <label for="sign-up-form-password">Password</label>
+            <input type="password" name="password" id="sign-up-form-password"/>
+          </div>
+          <div className="form-group">
+            <label for="sign-up-form-dob">Date of Birth</label>
+            <input type="date" name="dateOfBirth" id="sign-up-form-dob"/>
+          </div>
+          <div className="form-group">
+            <label>Notification Preference</label>
+            <label><input type="radio" name="notificationPreference"/> Email</label>
+            <label><input type="radio" name="notificationPreference"/> Phone</label>
+            <label><input type="radio" name="notificationPreference"/> None</label>
+          </div>
+          
+          <div className="form-group">
+            <label for="sign-up-form-bio">Write About Yourself</label>
+            <textarea name="biography" id="sign-up-form-bio" rows="3"></textarea>
+          </div>
+          <div className="form-group">
+            <div>
+              <label><input type="checkbox" name="agreeToTerms" value="true"/> I agree to the terms and conditions</label>
+            </div>
+            <div>
+              <label><input type="checkbox" name="subscribeToNewsLetter"/> Subscribe to newsletter</label>
+            </div>
+          </div>
+          <div>
+            <input type="submit" value="Sign Up"/>
+          </div>
+        </form>
+      </div>
+    </main>
+    <footer className="site-footer">
```

Comment:
Since the footer is common to all pages, we should move it to App.js (this is the component that defines the layout for all pages)

Reply:
@Pybite I want you to test the app as is. If you had done that you would see there is a double footer at the bottom.

You can `npm start` to see the app running, then you change the URL, add /signup

Reply:
at first I was having a bit of a issue starting it up but I got it working

Reply:
@Pybite yeah so you make sure no errors appear in npm start and in the browser (check browser console too)